assets/scripts/storage.js

```diff
@@ -0,0 +1,53 @@
+// Saves new goals
```

Comment:
This is raw JS, it should go in the build directory. Use Koala compiler to minify them and build into the assets directory.

Reply:
Resolved